static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -326,3 +326,28 @@ export function getDiseaseCommonName(data: GraphNodes): string {
     return formattedDiseaseName;
   }
 }
+/**
+ * Checks whether the stat var for medical condition of disease exists
+ * @param data
+ * @returns boolean indicating if desired stat var exists
+ */
+export function doesDiseasePrevalenceIDexist(data: GraphNodes): boolean {
+  // sets the default value of the boolean as false
+  if (!data) {
```

Comment:
probably want to check that data.nodes isn't empty?

Reply:
Right! Added a check for empty `data.nodes` and empty `data.nodes[0].neighbors`!